### PR TITLE
GUVNOR-3198: [Guided Decision Table] Visual indication where the current view of Decision Table is located

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/GUVNOR-3198

Demo

---

Part of an ensemble:
- https://github.com/AppFormer/uberfire/pull/808
- https://github.com/kiegroup/drools-wb/pull/583
- https://github.com/kiegroup/kie-wb-common/pull/1110

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/literal/LiteralExpressionEditorViewImplTest.java [hunk 1]
```diff
@@ -72,11 +72,11 @@ public void checkLayerConfiguration() {
         final Mediators m = view.getGridPanel().getViewport().getMediators();
         final List<IMediator> mediators = new ArrayList<>();
         m.iterator().forEachRemaining(mediators::add);
-        assertEquals(1,
+        assertEquals(2,
                      mediators.size());
-        assertTrue(mediators.get(0) instanceof RestrictedMousePanMediator);
+        assertTrue(mediators.get(1) instanceof RestrictedMousePanMediator);
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/literal/LiteralExpressionEditorViewImplTest.java]: Can you be absolutely confident this is always index ```1```?

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/literal/LiteralExpressionEditorViewImplTest.java [hunk 2]
```diff
@@ -72,11 +72,11 @@ public void checkLayerConfiguration() {
...
 
-        final RestrictedMousePanMediator mediator = (RestrictedMousePanMediator) mediators.get(0);
+        final RestrictedMousePanMediator mediator = (RestrictedMousePanMediator) mediators.get(1);
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/types/literal/LiteralExpressionEditorViewImplTest.java]: Can you be absolutely confident this is always index ```1```?

**Comment by karreiro** [comment]
Yes, @manstis.

The current implementation of the `LiteralExpressionEditorView`  extends the `BaseExpressionEditorViewImpl`, which has a final `GridLienzoPanel` being instantiated by the `GridLienzoPanel(width, height)` constructor. Finally, this constructor initializes the `GridLienzoScrollHandler` that adds the scroll mediator. Only then the tested `RestrictedMousePanMediator` is added by the `Mediators.push` (`lienzo-core:2.0.291-RELEASE`).

I'm not sure if I'm missing something. Am I?

--

Regarding renaming the method, it's done :-)

**Comment by karreiro** [comment]
Jenkins please retest this.

**Pull request merged**